Components/Bites/src/OgreApplicationContextQt.cpp

```diff
@@ -129,7 +135,27 @@ namespace OgreBites
         p.width = window->width();
         p.height= window->height();
 
-        p.miscParams["externalWindowHandle"] = std::to_string(size_t(window->winId()));
+        if (QGuiApplication::platformName() != "wayland") {
```

Comment:
Need to investigate whether other platform names may be relevant too. 
https://github.com/qt/qtwayland/blob/dev/README